class/defaults.yml

```
@@ -37,6 +37,27 @@ parameters:
           type: fluentd
           fluentd: {}
 
+    clusterLogForwarding:
+      enable: false
+      outputs: []
+      inputs: []
+      pipelines:
+        - name: audit-logs
+          inputRefs:
+            - audit
+          outputRefs:
+            - default
+        - name: infrastructure-logs
+          inputRefs:
+            - infrastructure
+          outputRefs:
+            - default
+        - name: application-logs
+          inputRefs:
+            - application
+          outputRefs:
+            - default
```

Maybe have the config here wrapped in a parameter `spec`

```suggestion
      spec:
        outputs: []
        inputs: []
        pipelines:
          - name: audit-logs
            inputRefs:
              - audit
            outputRefs:
              - default
          - name: infrastructure-logs
            inputRefs:
              - infrastructure
            outputRefs:
              - default
          - name: application-logs
            inputRefs:
              - application
            outputRefs:
              - default
```

---

Removed the `outputs` and `inputs`, but otherwise moved the parameters to the `spec` key.